Xamarin.Forms.Platform.WPF/FormsTextBox.cs

```diff
@@ -249,13 +249,15 @@ void OnTextChanged(object sender, System.Windows.Controls.TextChangedEventArgs t
 				string updatedText = Obfuscate(updatedRealText);
 				var savedSelectionStart = SelectionStart;
 
+				_internalChangeFlag = true;
```

Comment:
In this control class (it is very similar to the UWP one) there is a `DelayObfuscation` method that is not being used. I think this specific case could be fixed using that method. Could you try it?

Reply:
@jsuarezruiz done

There was some issues in DelayObfuscation with SelectionStart & SelectionLength when typing in beginning or middle of the text, but now it pretends to work correct =)